content/millennium/r4/individuals/related-person.md

```diff
@@ -47,6 +48,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `related-person-encounter` | [`reference`]       | Reference to the Encounter associated to the encounter level RelatedPerson.
  `relationship-level`       | [`CodeableConcept`] | The resource's relationship to either patient or encounter.
  `period`                   | [`Period`]          | A time period defined by a start and end date/time.
+ `relation`                 | [`CodeableConcept`]        | The related person's familial relationship to the patient.
```

Comment:
nit: The pipes should line up here

Reply:
done her https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR51